### PR TITLE
Add support for minimal border for tables

### PR DESCRIPTION
**Pull request by vmiklos**
Useful for markdown which has no borders by default. Based on what asciidoc does by default.

Sample output:

https://vmiklos.hu/blog/sw-zotero-plumbing.html

### CODE REVIEW

File: static/css/style.css [hunk 1]
```diff
@@ -19,6 +19,19 @@ hr {
...
+  border-collapse: collapse;
+}
+th,
+td {
+  padding: 4px;
+}
+table,
+th,
+td {
+  border-width: thin;
```
Comment (gfidente) [inline static/css/style.css:32]: couldn't border-width and border-style go together with padding on line 27?

File: static/css/style.css [hunk 2]
```diff
@@ -33,12 +46,12 @@ aside div#user_meta h2 {
 }
 aside div#user_meta p {
   margin: 0 0 15px 0;
-  font-size: .8rem;
+  font-size: 0.8rem;
```
Comment (gfidente) [inline static/css/style.css:49]: can you take css formatting changes out of this pull request and, if needed, submit those in a different PR?

**Review by gfidente — CHANGES_REQUESTED** [review]
I left some comments inline, seems good to me overall but please give those a look

File: static/css/style.css (hunk 1 above)
Comment (vmiklos) [reply static/css/style.css:32]: Seems indeed border-with and border-style is not needed for table (only for th and td). Done.

File: static/css/style.css (hunk 2 above)
Comment (vmiklos) [reply static/css/style.css:49]: Indeed, seems I have a lessc version that is different from the one used last time to generate .css from .less. I don't think this noise is worth to check in, so I just dropped them.

**Review by gfidente — APPROVED** [review]
thanks
